sale_blanket_order/wizard/create_sale_orders.py

```diff
@@ -183,7 +183,7 @@ class BlanketOrderWizardLine(models.TransientModel):
     _description = "Blanket order wizard line"
 
     wizard_id = fields.Many2one("sale.blanket.order.wizard")
-    blanket_line_id = fields.Many2one("sale.blanket.order.line")
+    blanket_line_id = fields.Many2one("sale.blanket.order.line", store="True")
```

Comment:
What's the reason for this? The field is stored by default, you don't need to add it the attribute`store`.

Reply:
This is a fix I copied from PR #1608 [^1] after a lot of troubleshooting. If this is not set to `True` the wizard cannot persist the lines when confirming. It will be reset and the wizard lines are empty. I was not able to investigate further, but I assume this is related to some change in v14. Also note that this is a transient model.

[^1]: https://github.com/OCA/sale-workflow/pull/1608/files#diff-2565286591068d712b8e7b02a8c2580442f249c417077cbd482b6c1b3b6403b4